torch/_dynamo/symbolic_convert.py

```diff
@@ -610,10 +626,40 @@ def inline_user_function_return(self, fn, args, kwargs):
         """
         state = self.copy_graphstate()
         try:
-            result = InliningInstructionTranslator.inline_call(self, fn, args, kwargs)
+            result, resume_at_insts = InliningInstructionTranslator.inline_call(
+                self, fn, args, kwargs
+            )
             self.output.guards.update(fn.guards)
-            return result
-        except Exception:
+            if resume_at_insts is None:
+                return result
+            # An inner inlining translator has a graph break
+            # Call into its resume_at function, before running our
+            # our resume_at instructions.
+            assert result is None
+            resume_at_insts, push = resume_at_insts
+            new_co_names = list(self.output.global_scope)
```

Comment:
Actually it goes beyond this, output is designed for a single function context. We need to be able to compile with the inlined context.